test/extractor/AssignmentRuleExtractor.test.ts

```diff
@@ -223,23 +219,16 @@ describe('AssignmentRuleExtractor', () => {
       element.patternRatio.numerator.system = 'http://other-units.org';
       element.patternRatio.numerator.unit = 'centimeters';
       const assignmentRules = AssignmentRuleExtractor.process(element);
-      const expectedNumeratorQuantity = new ExportableAssignmentRule('valueRatio.numerator');
-      expectedNumeratorQuantity.value = new fshtypes.FshCode(
-        'cm',
-        'http://other-units.org',
-        'centimeters'
-      );
-      const expectedNumeratorValue = new ExportableAssignmentRule('valueRatio.numerator.value');
-      expectedNumeratorValue.value = 5;
-      const expectedDenominatorRule = new ExportableAssignmentRule('valueRatio.denominator');
-      expectedDenominatorRule.value = new fshtypes.FshQuantity(
-        1,
-        new fshtypes.FshCode('s', 'http://unitsofmeasure.org')
+      const expectedRule = new ExportableAssignmentRule('valueRatio');
+      expectedRule.value = new fshtypes.FshRatio(
+        new fshtypes.FshQuantity(
+          5,
+          new fshtypes.FshCode('cm', 'http://other-units.org', 'centimeters')
+        ),
+        new fshtypes.FshQuantity(1, new fshtypes.FshCode('s', 'http://unitsofmeasure.org'))
       );
-      expect(assignmentRules).toHaveLength(3);
-      expect(assignmentRules[0]).toEqual<ExportableAssignmentRule>(expectedNumeratorQuantity);
-      expect(assignmentRules[1]).toEqual<ExportableAssignmentRule>(expectedNumeratorValue);
-      expect(assignmentRules[2]).toEqual<ExportableAssignmentRule>(expectedDenominatorRule);
+      expect(assignmentRules).toHaveLength(1);
+      expect(assignmentRules[0]).toEqual<ExportableAssignmentRule>(expectedRule);
```

Comment:
Similar to the test I commented on above, I'm not sure we really need this test.